### ISSUE TITLE
refresh coverage lines

### ISSUE THREAD

**Issue by juanpabloaj** [alfredodeza/coveragepy.vim#6]
I run coverage 

```
coverage run setup.py test
```

in vim, open coveragepy

```
 :Coveragepy show
```

ok. Then I modifify the code and run coverage again (in other terminal, without close vim)

```
coverage run setup.py test
```

and hide and show Coveragepy

```
 :Coveragepy show
 :Coveragepy show
```

but, the coverages lines don't are updated.

**Comment by eprigorodov** [comment]
This function, refreshing coverage marks without opening report buffer, still seems to be desirable for work on completing test coverage. My typical cycle is: scroll to some uncovered lines, write corresponding unit tests in adjacent split buffer, re-run unit tests under `coverage.py`. And then I would like to see newly covered lines green and move to the next batch. Using `:Coveragepy report` for that requires switching to report buffer and closing it, which is just a waste of time on each cycle.

How about adding `:Coveragepy reload` command which would invoke `coverage report` and update coverage marks without opening report window? I already have a patch.

**Comment by alfredodeza** [comment]
@eprigorodov I like your solution :)

**Comment by eprigorodov** [comment]
And I like your plugin, it just works and does what was needed. Please have a look at PR #17. Thank you :)

**Comment by alfredodeza** [comment]
Fixed with PR #17
